src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.ts

```diff
@@ -209,7 +180,8 @@ export async function suggest(
     astContext.type === 'expression' ||
     (astContext.type === 'option' && astContext.command?.name === 'join') ||
     (astContext.type === 'option' && astContext.command?.name === 'dissect') ||
-    (astContext.type === 'option' && astContext.command?.name === 'from')
+    (astContext.type === 'option' && astContext.command?.name === 'from') ||
+    (astContext.type === 'option' && astContext.command?.name === 'enrich')
```

Comment:
I know that this is temporary but maybe it makes sense for now to create an array with the converted commands and have one check here

Reply:
Good point, I can do this!